nomos-cli/src/cmds/disseminate/mod.rs

```diff
@@ -67,7 +66,7 @@ impl Disseminate {
                     send_blob: Settings {
                         payload: Arc::new(Mutex::new(payload_rx)),
                         timeout,
-                        da_protocol,
+                        kzgrs_settings: KzgrsSettings::default(),
```

Comment:
were this setting using the `toeplitz_cache`? We probably want to use that feature (as it is much faster).